src/controller/python/matter/clusters/Objects.py

```
@@ -34376,6 +34387,42 @@ def descriptor(cls) -> ClusterObjectDescriptor:
             numberOfSchedules: 'uint' = 0
             scheduleTypeFeatures: 'uint' = 0
 
+        @dataclass
+        class SensorScheduleTransitionStruct(ClusterObject):
+            @ChipUtility.classproperty
+            def descriptor(cls) -> ClusterObjectDescriptor:
```

Pylint (4.0.7)

[error] Method 'descriptor' should have "self" as first argument

(E0213)